build/import/Packages.targets

```diff
@@ -2,6 +2,7 @@
 <Project>
 
   <PropertyGroup>
+    <!-- NOTE: Keep in-sync with NuGet.config at repo root -->
```

Comment:
Just an idea, but we could add a unit test to verify these are in sync.

Reply:
I think there is a way to eliminate having 2 of them. I'd rather opt for that strategy. Added an issue here: https://github.com/dotnet/project-system/issues/7870

Reply:
That'd be even better.